build-octotiger.sh

```diff
@@ -7,15 +7,14 @@ set -ex
     ${OCT_WITH_KOKKOS_SCALAR:?} ${CUDA_SM:?} ${OCT_ARCH_FLAGS:?}
 
 
-DIR_SRC=${SOURCE_ROOT}/octotiger
-DIR_BUILD=${INSTALL_ROOT}/octotiger/build
+DIR_SRC=${SOURCE_ROOT}/octotiger-kokkos
+DIR_BUILD=${INSTALL_ROOT}/octotiger/build-kokkos
```

Comment:
I don't particularly like this change (breaks some of my scripts and my own muscle memory :-) )

Is there a specific reason for it?